lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -60,7 +60,7 @@
                     })
                 };
             }
-
+            var tabSection = {};
```

Comment:
The plural name would suit better for this variable, like `tabSections` or similar.